packages/playground/src/pages/Dropin/session.js

```diff
@@ -46,6 +46,28 @@ export async function initSession() {
                 // billingAddress config:
                 billingAddressRequired: true,
                 billingAddressMode: 'partial'
+            },
+            paypal: {
+                isExpress: true,
+
+                onShippingAddressChange: async (data, actions, component) => {
+                    const patch = {
+                        sessionId: session.id,
+                        paymentData: component.paymentData,
+                        amount: {
+                            currency: 'USD',
+                            value: 29900
+                        }
+                    };
+
+                    if (data.shippingAddress.countryCode === 'US') {
+                        const { paymentData } = await patchPaypalOrder(patch);
+                        component.updatePaymentData(paymentData);
+                        return;
+                    }
+
+                    return actions.reject();
+                }
```

Comment:
Would it also be good to have an example of the `onShippingOptionsChange` callback? Or is the process the same?

Reply:
I focused mainly on documenting this in the Storybook playground. I should remove this code from here